README.md

```diff
@@ -42,6 +43,30 @@ Prequisites:
   redis-cli ping  # Should respond with PONG
   rake db:setup
   ```
+Ubuntu:
+- Update system
+  ```shell
+  sudo apt update
+  ```
+- Install redis server
+  ```shell
+  sudo apt install redis-server
+  ```
+- Update configuration
+  ```shell
+  sudo sh -c 'echo "supervised systemd" >> /etc/redis/redis.config'
+  ```
+- Restart running service
+  ```shell
+  sudo systemctl restart redis.service
+  ```
+- Configure and verify redis
+  ```shell
+  redis-cli CONFIG SET dir /tmp/
+  redis-cli CONFIG SET dbfilename temp.rdb
+  redis-cli ping  # Should respond with PONG
+  rake db:setup
```

Comment:
Do you need to have the command `redis-server` included in the configuration for Ubuntu, similarly to the instructions in OSX?

Reply:
I second this question - I believe you would still need to include running `redis-server` in the instructions?

Reply:
what is the purpose of running redis-cli pong ? Is it to check whether the server is alive?

Reply:
After we update the redis configuration file to specify using `systemd` to manage redis, we run the command:
`sudo systemctl restart redis.service`, which restarts the redis systemd service, which starts running the redis-server. So for these instructions, we don't need an additional command to run `redis-server` in the background in the same manner as the OSX instructions, since `systemd` is managing redis and since we start the redis server with that `systemctl` command. And @sandipa12, yes the pong is just to ensure that the server is running.